qemu/tests/cfg/rh_kernel_update.cfg

```diff
@@ -25,8 +25,8 @@
     RHEL.7:
         kernel_deps_pkgs += " kmod hmaccalc"
         upgrade_pkgs += " qemu-guest-agent"
-    RHEL.8:
-        kernel_pkgs += " kernel-core kernel-modules"
+    RHEL.8, RHEL.9:
```

Comment:
Suggest `! RHEL.5, RHEL.6, RHEL.7:`, more friend for new version in the future.

Reply:
Indeed,thanks!